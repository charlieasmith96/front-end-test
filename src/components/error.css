.error-message {
    width: 250px;
    margin: 0 auto;
    color: indianred;
    text-align: center;
    padding-bottom: 10px;
}