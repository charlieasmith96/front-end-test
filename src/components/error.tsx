import React from 'react';
import './error.css';

interface ErrorProps {
    errorMessage?: string | null
    className?: string
}

export const Error = (props: ErrorProps) => {
    return (
        <div className={`error-message ${props.className}`}>
            {props.errorMessage}
        </div>
    )
}