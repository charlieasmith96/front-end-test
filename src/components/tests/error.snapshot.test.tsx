import React from 'react';
import renderer from 'react-test-renderer'
import { Error } from '../error';

it('should render header correctly', () => {
    const tree = renderer.create(<Error errorMessage="test-message"/>)
    expect(tree).toMatchSnapshot();
})