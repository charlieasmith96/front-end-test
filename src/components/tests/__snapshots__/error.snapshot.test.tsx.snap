// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header correctly 1`] = `
<div
  className="error-message undefined"
>
  test-message
</div>
`;
